<?xml version="1.0" encoding="utf-8"?>
<launch>
  <!-- Generate sinusodial wave -->
  <node name="sinusoidal_wave_generator" pkg="bulldog_mujoco_ros" type="sinusoidal_wave_generator.py" output="screen">
    <param name="command_topic" value="/ur5e_robotiq_2f_85_manipulator_controller/command" />
    <param name="command_joint" value="robot0_elbow_joint" />
    <param name="wave_mag" value="1.5708" />
    <param name="wave_offset" value="0.0" />
    <param name="wave_freq" value="25" />
  </node>
</launch>

<!--
  robot0_shoulder_pan_joint: {p: 5000.0, i: 0.01, d: 180.0, i_clamp: 1.0}
  robot0_shoulder_lift_joint: {p: 48000.0, i: 0.0, d: 200.0, i_clamp: 1.0}
  robot0_elbow_joint: {p: 48000.0, i: 0.0, d: 5.0, i_clamp: 1.0}
  robot0_wrist_1_joint: {p: 4500.0, i: 0.0, d: 0.0, i_clamp: 1.0}
  robot0_wrist_2_joint: {p: 4000.0, i: 0.1, d: 2.0, i_clamp: 1.0}
  robot0_wrist_3_joint: {p: 450.0, i: 0.0, d: 0.5, i_clamp: 1.0}
-->
